migrator.sh

```diff
@@ -347,6 +347,47 @@ decode_auth() {
   fi
 }
 
+##
+# Get all the tags for a name at the target repository
+#
+# Usage: query_tags_at_target <image_name>
+# Return: json-list of all tags in the form ["latest", "dev", "123"]. If the request fails, the list will be empty
+##
+query_tags_at_target() {
+    IMAGE="${1}"
+
+    TAGS_URL="https://${V2_REGISTRY}/v2/${IMAGE}/tags/list"
+
+    TAGS_RETRIEVAL_POSSIBLE=$(curl -s -o /dev/null -w "%{http_code}" ${TAGS_URL} --user ${V2_USERNAME}:${V2_PASSWORD})
```

Comment:
I'd suggest adding `${INSECURE_CURL}` to allow curl to function with self-signed/untrusted certs if the user wants to.